src/elm/Page/Shop/Editor.elm

```diff
@@ -453,6 +464,38 @@ viewForm shared balances imageStatus isEdit isDisabled form =
         ]
 
 
+viewConfirmDeleteModal : (String -> String) -> Html Msg
+viewConfirmDeleteModal t =
+    div [ class "modal container" ]
+        [ div [ class "modal-bg", onClick ClickedDeleteCancel ] []
+        , div [ class "modal-content" ]
+            [ div [ class "w-full" ]
+                [ p [ class "text-2xl font-medium mb-4" ]
+                    [ text (t "shop.delete_modal.title") ]
+                , button [ onClick ClickedDeleteCancel ]
+                    [ Icons.close "absolute fill-current text-gray-400 top-0 right-0 mx-8 my-4" ]
+                , p [ class "text-body w-full font-sans mb-10" ]
+                    [ text (t "shop.delete_modal.body") ]
+                ]
+            , div [ class "w-full md:bg-gray-100 md:flex md:absolute rounded-b-lg md:inset-x-0 md:bottom-0 md:p-4 justify-center" ]
+                [ div [ class "flex" ]
+                    [ button
+                        [ class "flex-1 block button button-secondary mb-4 button-lg w-full md:w-40 md:mb-0"
+                        , onClick ClickedDeleteCancel
+                        ]
+                        [ text (t "shop.delete_modal.cancel") ]
+                    , div [ class "w-8" ] []
+                    , button
+                        [ class "flex-1 block button button-primary button-lg w-full md:w-40"
+                        , onClick ClickedDeleteConfirm
+                        ]
```

Comment:
So the modal when small should have block buttons just as on Figma.

here it is how its displayed on Safari:

<img width="497" alt="Screen Shot 2020-04-03 at 11 49 52" src="https://user-images.githubusercontent.com/1082127/78373635-3bb31d00-75a1-11ea-8cf2-c4a7d40e2314.png">

Here it is how Figma instructs:

<img width="558" alt="Screen Shot 2020-04-03 at 11 50 51" src="https://user-images.githubusercontent.com/1082127/78373746-5f766300-75a1-11ea-9381-dad48e8edb26.png">

Link for the Figma file: https://www.figma.com/file/67j4KffwVAh9lB9XMQ4vfS/Modals?node-id=0%3A1

Reply:
Oh alright! I hadn't seen those modals designs, the issue only linked to the big one! I didn't even consider that! I'll see to it then!